test/appframework/aws/s1/appframework_aws_suite_test.go

```diff
@@ -50,7 +50,7 @@ var (
 	currDir, _            = os.Getwd()
 	downloadDirV1         = filepath.Join(currDir, "s1appfwV1-"+testenv.RandomDNSName(4))
 	downloadDirV2         = filepath.Join(currDir, "s1appfwV2-"+testenv.RandomDNSName(4))
-	downloadDirPVTestApps = filepath.Join(currDir, "s1appfwPVTestApps-"+testenv.RandomDNSName(4))
+	downloadDirPVTestApps = filepath.Join(currDir, "c3appfwPVTestApps-"+testenv.RandomDNSName(4))
```

Comment:
NIT: should the names be `m4appfwPVTestApps `. This is the m4 directory

Reply:
Indeed, bad copy-paste